packages/node_modules/@ciscospark/spark-core/test/unit/spec/lib/batcher.js

```diff
@@ -158,7 +158,7 @@ describe(`spark-core`, () => {
 
           const promises = [];
           // eslint-disable-next-line no-unmodified-loop-condition
-          for (let i = 0; i < BATCHER_MAX_CALLS + 1; i++) {
```

Comment:
use `<=` instead of `+1`. it's slightly more efficient and in keeping with the other place where you already changed to `<=`